modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/endpoint/SchedulerRestService.java

```diff
@@ -32,6 +32,7 @@
 import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
 import static org.opencastproject.capture.CaptureParameters.AGENT_REGISTRATION_TYPE;
 import static org.opencastproject.capture.CaptureParameters.AGENT_REGISTRATION_TYPE_ADHOC;
+import static org.opencastproject.mediapackage.MediaPackageElement.Type.Catalog;
```

Comment:
Your additions seem not to require this import and line 50 already imports a type of the same name (` import org.opencastproject.mediapackage.Catalog;`)

Why have you added this import?

Reply:
That import must be an artifact but I'm surprised it didn't throw an "unused import" checkstyle error. I've removed that import.